.github/workflows/backend-api-push-to-main.yml

```diff
@@ -144,3 +148,17 @@ jobs:
           signing-profile-name: ${{ secrets.BACKEND_API_DEV_SIGNING_PROFILE }}
           template-file: .aws-sam/build/template.yaml
           working-directory: backend-api
+
+      - name: Configure AWS credentials
```

Comment:
Should we move these into a separate job `build-and-upload-sam-artifact-to-build`?